src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceActions.ts

```diff
@@ -162,7 +242,9 @@ export var hideActionDirective = (
 ) => {
     return {
         restrict: "E",
-        template: "<a class=\"{{class}}\" href=\"\" data-ng-click=\"hide();\">{{ 'TR__HIDE' | translate }}</a>",
+        transclude: true,
+        template: "<a class=\"{{class}}\" href=\"\" data-ng-click=\"hide();\"><ng-transclude></ng-transclude>" +
+            "{{ 'TR__HIDE' | translate }}</a>",
```

Comment:
Adding transclusion to all of these is overly complex IMHO. Why not add the icons to the template directly?